types/universal-router/generateUrls.d.ts

```diff
@@ -0,0 +1,14 @@
+import UniversalRouter, { Params } from ".";
+
+export interface generateUrlsOptions {
+    /**
+     * Provide a function to stringifyQueryParams option to generate URL with query string from unknown route params.
+     */
+    stringifyQueryParams?: (params: Params) => string;
+    [_: string]: any;
```

Comment:
Oh I completely forgot to mention this one. As I understand from [universal-router API](https://github.com/kriasoft/universal-router/blob/master/docs/api.md), there are only two options for `generateUrls`: `stringifyQueryParams` and `encode` from `path-to-regexp`. So if you include this library, you would be able to inherit `PathFunctionOptions` interface that provides `encode` option:
```typescript
export interface Options extends PathFunctionOptions {
  stringifyQueryParams?: (params: Params) => string;
}
```
Please correct me if I'm wrong.